contacts_app/lib/helperFunctions.dart

```diff
@@ -0,0 +1,69 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_sms/flutter_sms.dart';
+import 'package:url_launcher/url_launcher.dart';
+import 'messagingDialog.dart';
+
+class HelperFunctions {
+  static Future<void> initiateMessage(
+      BuildContext context, String message, String recipient) async {
+    final String _result =
+        await sendSMS(message: message, recipients: <String>[recipient])
+            .catchError((dynamic onError) {
+      print(onError);
```

Comment:
Should the final code have print statements?

Reply:
Thought it would be a good log to keep? I could remove it though if you think we shouldnt have it